lesson_2/sortDesc.js

```diff
@@ -0,0 +1,7 @@
+function compareNumbers(a, b) {
+  return b - a;
+}
+
+const sortDesc = function(numbers) {
+  return numbers.sort(compareNumbers)
+}
```

Comment:
Такую простую _callback_ функцию можно не выносить отдельно, а определить сразу в аргументах:

```suggestion
const sortDesc = function(numbers) {
  return numbers.sort((a, b) => b - a)
}
```